test/morgan.js

```diff
@@ -1067,7 +1219,7 @@ describe('morgan()', function () {
           var cb = after(2, function (err, res, line) {
             if (err) return done(err)
             var masked = line.replace(/\x1b\[(\d+)m/g, '_color_$1_')
-            assert.equal(masked, '_color_0_GET / _color_0_- _color_0_- ms - -_color_0_')
```

Comment:
Please put back the leading color resets. It was added because there were cases when the string would print, but the previous thing did not reset it's color, so the beginning would end up being a random color.